src/common/src/types/to_text.rs

```diff
@@ -14,33 +14,64 @@
 
 use std::fmt::Write;
 
-use super::{DatumRef, ScalarRefImpl};
+use chrono::{TimeZone, Utc};
+
+use super::{DataType, DatumRef, ScalarRefImpl};
 
 // Used to convert ScalarRef to text format
 pub trait ToText {
+    // to_text impl to_text_with_type using default type of ScalarRefImpl.
```

Comment:
Use `///` and make it clear that developers should prefer `to_text_with_type`.

Reply:
why prefer `to_text_with_type`🤔 if developer want to convert to the default type, he can just use `to_text()`

Reply:
In what situation can a developer be confident that `i64` is bigint rather than timestamp with time zone? The default is biased towards bigint without a clear reason.